packages/edit-site/src/components/preferences-modal/index.js

```diff
@@ -52,10 +51,10 @@ export default function EditSitePreferencesModal() {
 	};
 
 	const turnOffDistractionFree = () => {
-		setPreference( 'core/edit-site', 'distractionFree', false );
+		setPreference( 'core', 'distractionFree', false );
 	};
 
-	const sections = useMemo( () => [
```

Comment:
Why remove the `useMemo`?

Reply:
Because it was useless (it had no dependencies) so it's like there's no useMemo to start with.